ClothesApp/WebAPI/Controllers/ReviewsController.cs

```diff
@@ -0,0 +1,57 @@
+using Application.Dtos.Reviews;
+using Application.Interfaces.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace WebAPI.Controllers;
+
+[ApiController]
+[Route("api/reviews")] 
+public class ReviewsController : ControllerBase
```

Comment:
You added endpoint to get all reviews all over the place, but two main use cases are to get all my reviews (by user id) and get product reviews (by product id)